src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -38,19 +38,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String requestEndTime = request.getParameter(Constants.END_TIME_PARAM);
     String timeZone = request.getParameter(Constants.TIME_ZONE_PARAM);
 
-    //create dates from input
-    Date startDate = createDateTime(requestStartDate.substring(0, 4),
+    //create dates with times from input for time verification
```

Comment:
Please add a space between // and the start of the sentence. 

Nit: Use sentence case.

Reply:
Same for other locations in this PR.